samples/idea-latest.yaml

```diff
@@ -0,0 +1,36 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: false
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/vitaliy-guliy/web-nodejs-sample.git"
+            # origin: "https://github.com/che-samples/web-nodejs-sample.git"
+          checkoutFrom:
+            revision: "devfilev2"
+    components:
+      - name: dev
+        container:
+          image: quay.io/devfile/universal-developer-image:latest
+#        attributes:
+#          controller.devfile.io/merge-contribution: true
+    commands:
+      - id: say-hello
+        exec:
+          component: dev
+          commandLine: echo "Hello from $(pwd)"
+          workingDir: ${PROJECT_SOURCE}/app
+  contributions:
+    - name: che-idea
+      uri: https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/che-incubator/che-idea/latest/devfile.yaml
+#      components:
+#        - name: che-idea-runtime-description
+#          container:
+#            env:
+#              - name: CODE_HOST
+#                value: 0.0.0.0
```

Comment:
Can be dropped for `che-idea`, as it currently listens on `0.0.0.0` by default (this may be an issue in Che, though, as it may bypass the gateway proxy).

Reply:
Removed